social.py

```diff
@@ -2,7 +2,7 @@
 import requests
 from bs4 import BeautifulSoup
 from pathlib import Path
-
+import base64
```

Comment:
Hey @akshitdevops,

Is the import base64 really necessary? I don't see it being used anywhere.

Reply:
Hey,
Please check line number 109 of the same file.